SpaceDash/Controller/ViewControllers/RocketViewController.swift

```diff
@@ -0,0 +1,261 @@
+        return v
+    }()
+    
+    public let latestLaunchMetric:CustomMetricCard = {
+        let v = CustomMetricCard()
+        v.title.text = "Latest Launch"
+        v.subTitle.text = "Sep 27, 2020 at 8:13PM"
+        return v
+    }()
+    
+    public let firstLaunchMetric:CustomMetricCard = {
+        let v = CustomMetricCard()
+        v.title.text = "First Launch"
+        v.subTitle.text = "Jan 6, 2018"
+        return v
+    }()
+    
+    public let engineAndThrustMetric:CustomMetricCard = {
+        let v = CustomMetricCard()
+        v.title.text = "Engine, Thrust"
+        v.subTitle.text = "Merlin 1D+, 914 KN"
+        return v
+    }()
+    
+    public let descriptionMetric:CustomMetricCard = {
+        let v = CustomMetricCard()
+        v.title.text = "Description"
+        v.subTitle.text = "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book."
+        v.subTitle.numberOfLines = 0
+        return v
+    }()
+    
+    public lazy var readMoreBtn:UIButton = {
+        let btn = UIButton()
+        btn.translatesAutoresizingMaskIntoConstraints = false
+        btn.setTitle("ReadMore", for: .normal)
+        btn.setTitleColor(.white, for: .normal)
+        btn.titleLabel?.font = UIFont.systemFont(ofSize: 14, weight: .bold)
+        btn.backgroundColor = .black
+        btn.layer.cornerRadius = CGFloat(15)
+        btn.addTarget(self, action: #selector(readmoreBtnPressed), for: .touchUpInside)
+        return btn
+    }()
+    
+    let bottomBlackView:UIView = {
+        let v = UIView()
+        v.translatesAutoresizingMaskIntoConstraints = false
+        v.backgroundColor = .black
+        return v
+    }()
+    
+    let pathView:UIView = {
+        let v  = UIView()
+        v.translatesAutoresizingMaskIntoConstraints = false
+        v.backgroundColor = .white
+        v.layer.cornerRadius = 35
+        v.layer.maskedCorners = [.layerMinXMaxYCorner, .layerMaxXMaxYCorner]
+        return v
+    }()
+    
+    /// Creating instance of CustomPopUpView
+    lazy var popUpView:CustomPopUpView = {
+        let v = CustomPopUpView()
+        v.delegate = self
+        v.translatesAutoresizingMaskIntoConstraints = false
+        return v
+    }()
+    
+    let overlayView:UIView = {
+        let v = UIView()
+        v.translatesAutoresizingMaskIntoConstraints = false
+        v.backgroundColor = UIColor(white: 0, alpha: 0.3)
+        return v
+    }()
+    
+    /// For dynamic height for popUpCard
+    var popUpViewCardHeight:CGFloat?
+    
+    /// Inorder to animate popUpView from bottom we need to change its bottom constraints, so assigning bottom constraints separately.
+    var popUpBottomConstraint:NSLayoutConstraint?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
```

Comment:
We are using a custom colour named DashCream for white. Use that colour instead of white everywhere.